raiden-ts/src/raiden.ts

```diff
@@ -111,6 +110,11 @@ export class Raiden {
    */
   public readonly transfers$: Observable<RaidenTransfer>;
 
+  /** RaidenConfig object */
+  public config!: RaidenConfig;
```

Comment:
Should we provide a callback'ish way as well? I guess we haven't decided yet in #357

Reply:
Yes, I was thinking about it, I really want to standardize these interfaces, but didn't want to make this PR more complex, so just used the same pattern we already have for `state$`, `channels$`, `transfers$` and so.. I'd say we should keep it like that for now, but I'm thinking on something like making `Raiden` an `EventEmitter` derived class and emitting these as events in a more interoperable way.